clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1630,6 +1635,52 @@ def GetMemberOp : CIR_Op<"get_member"> {
   let hasVerifier = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// VecElemOp
+//===----------------------------------------------------------------------===//
+
+def VecElemOp : CIR_Op<"vec_elem", [Pure,
+  TypesMatchWith<"type of 'result' matches element type of 'vec'",
```

Comment:
How about we prefix vector related operations with `cir.vec`, so this would be `cir.vec.elem` or `cir.vec.elt`? This goes in hand with other things like `cir.libc` or `cir.std`, etc.

Reply:
Makes sense.  But after thinking about this some more, I want to change it to `cir.vec.extract`.  That describes more precisely what this operation does.

Reply:
sounds good to me!

Reply:
Done.